.env.example

```diff
@@ -3,8 +3,8 @@ ENVIRONMENT="staging"
 
 # URLs & Paths
 DEFAULT_SITE_URL=""
-ASSET_BASE_URL=""
-ASSET_BASE_PATH=""
+ASSET_BASE_URL=""      # Full URL to the folder with the static and media assets. Typically same as the site url on dev, and CDN base url on production. eg. katana.local/project/ or cdn.project.com/
```

Comment:
The examples themselves are not _full_ URLs. They are missing a URL scheme such as `http://` which is expected to be part of the value. Ensure that the example provided is a valid input.